roles/metrics_server/defaults/main.yaml

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+metrics_server_helm_repository_name: metrics-server
+metrics_server_helm_repository_url: https://kubernetes-sigs.github.io/metrics-server
+metrics_server_helm_chart_name: metrics-server
+metrics_server_helm_chart_version: 3.12.2
+
+metrics_server_helm_release_namespace: kube-system
+metrics_server_helm_release_name: metrics-server
+
+metrics_server_helm_values: {}
+metrics_server_node_selector: {}
```

Comment:
I think this variable is not used, what do you think, should we incorporate it as a role variable by default or should we have users use the node selector in the helm values?

Reply:
Good point.